integration-tests/tests/add-component-to-application.spec.ts

```diff
@@ -16,15 +16,9 @@ describe('Create Components using the UI', () => {
     'https://github.com/nodeshift-starters/devfile-sample.git',
   ];
   const componentNames = ['java-quarkus', 'go', 'go-component', 'nodejs'];
```

Comment:
wondering if we still need to use these pre-defined component names when they now have randomly generated unique names instead. wdyt?

Reply:
That's a good point, I missed that.
> is that fine?

Yes sure